modules/ob-canceling-a-build-run.adoc

```diff
@@ -0,0 +1,21 @@
+// This module is included in the following assembly:
+//
+// builds/configuring-build-runs.adoc
+
+:_content-type: CONCEPT
+[id="ob-canceling-a-build-run_{context}"]
+= Cancel a build run
```

Comment:
In the doc guidelines, its is said to use the gerund form in headings (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings) 
There are a few titles in this doc set that need to be updated to use the gerund form. 

```suggestion
= Canceling a build run
```

Reply:
@shipsing I'm still merge-reviewing this, but I wanted to get a comment on the titles in sooner.

Per Kelly's comment above, it is true we want gerunds in titles for procedures, but I think a problem I'm noticing in the latest version of the PR is that while the "Cancel / canceling a build run" title looks like a procedure at first glance, the module currently has `:_content-type: REFERENCE` (instead of `PROCEDURE`), and then the body content doesn't have a `.Procedure` with actual steps.

From above cited repo guidelines:
> Avoid using gerunds in concept titles. "About <concept>" is a common concept module title.

Unfortunately the guidelines don't mention explicitly either way about _reference_ titles, but we do usually save the gerunds for procedures in general. Take for another example the "Defining the service account" section/module elsewhere: the title looks like a procedure but it's a reference module with no procedure/steps.

Maybe most of the "Defining..." titles could be reworked to something like:

"Defining the build reference" -> "Build reference definition"
"Defining the build specification" -> "Build specification definition"
"Defining parameter values for a build run" -> "Parameter values for a build run"
"Defining the service account" -> "Service account definition"

So perhaps the overall comment is: check to see if the titles make sense / follow guidelines for being a reference module. Alternatively, if you want to keep the gerund, possibly rework the module to become a procedure.

I'll continue reading through the PR and will remove the `merge-review-in-progress` when complete. Thank you!

Reply:
@adellape I am not keen to make the sections look like a procedure because the procedure is very simple, you just need to create a `BuildRun` resource YAML file and apply it to the OpenShift Container Platform cluster. 
In this assembly, the focus is on different aspects associated with configuring a build run.  like, what all the parameters you can configure in your `BuildRun` resource YAML file and if you define the value for the same parameter in both `Build` and `BuildRun` CR, then what would be the preference.

So, I am making adjustments in the heading.